lib/bap_image/bap_memory.ml

```diff
@@ -464,6 +464,12 @@ let () =
   KB.collect slot label >>|? fun mem ->
   Some (Addr.to_bitvec (min_addr mem))
 
+let () =
+  let open KB.Syntax in
+  KB.promise Theory.Label.bytes @@ fun label ->
+  KB.collect slot label >>|? fun {data; off; size} ->
+  Some (Bigstring.to_string ~pos:off ~len:size data)
+
```

Comment:
Since we now moved it to the property of the semantics itself, we have to provide the semantics instead, 

```suggestion
let () =
  KB.promise Theory.Semantics.slot @@ fun label ->
  let+ {data; off; size} = label-->?Memory.slot in
  KB.Value.put Theory.Semantics.code Insn.empty @@
  Option.some @@
  Bigstring.to_string ~pos:off ~len:size data
```

Reply:
👍